code/modules/client/preferences.dm

```diff
@@ -1392,37 +1453,50 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 							second_limb = "r_hand"
 						if("Left Foot")
 							limb = "l_foot"
-							third_limb = "l_leg"
+							if(species != "Machine")
+								third_limb = "l_leg"
 						if("Right Foot")
 							limb = "r_foot"
-							third_limb = "r_leg"
+							if(species != "Machine")
+								third_limb = "r_leg"
 						if("Left Hand")
 							limb = "l_hand"
-							third_limb = "l_arm"
+							if(species != "Machine")
+								third_limb = "l_arm"
 						if("Right Hand")
 							limb = "r_hand"
-							third_limb = "r_arm"
+							if(species != "Machine")
+								third_limb = "r_arm"
 
 					var/new_state = input(user, "What state do you wish the limb to be in?") as null|anything in valid_limb_states
 					if(!new_state) return
 
 					switch(new_state)
 						if("Normal")
+							if(limb == "head")
+								m_style = "None"
+								h_style = random_hair_style(gender, species)
+								f_style = facial_hair_styles_list["Shaved"]
 							organ_data[limb] = null
 							rlimb_data[limb] = null
 							if(third_limb)
 								organ_data[third_limb] = null
 								rlimb_data[third_limb] = null
 						if("Amputated")
-							organ_data[limb] = "amputated"
-							rlimb_data[limb] = null
-							if(second_limb)
-								organ_data[second_limb] = "amputated"
-								rlimb_data[second_limb] = null
+							if(!no_amputate)
+								organ_data[limb] = "amputated"
+								rlimb_data[limb] = null
+								if(second_limb)
+									organ_data[second_limb] = "amputated"
+									rlimb_data[second_limb] = null
 						if("Prothesis")

```

Comment:
Fixed this typo in #3943, supposed to be Prosthesis

prothesis proboscis PNODSFPOGNGINS[D